packages/ui/src/fonts/ClashDisplay/style.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
```

Comment:
hm, I'm inclining towards something like this
https://github.com/react-to/react-to-styled/blob/develop/packages/essentials/styles/global.ts
So that we can import this in our Storybook and also our App.
But let me know if you have any trouble with this implementation.

Reply:
@aleksaToljic done